tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html

```diff
@@ -534,6 +584,24 @@ <h3>No profile data was found.</h3>
       // Otherwise, remove the seperator, e.g. "host1_" => "host1".
       return host.slice(0, -1);
     },
+    _computeTopLevelState: function(inColab, dataNotFound, progress) {
+      if (inColab)
+        return "IN_COLAB";
+      if (dataNotFound)
+        return "DATA_NOT_FOUND";
+      if (!progress || progress.value < 100)
```

Comment:
I think we can now remove `_isNotComplete()` since its logic was inlined here and it's not called anywhere else

Reply:
Good point; done.